app/services/foreman_discovery/host_converter.rb

```diff
@@ -51,7 +60,6 @@ def self.unused_ip_for_host(host, new_subnet = nil, new_subnet6 = nil)
       interface.subnet6 = new_subnet6 if new_subnet6
       interface.ip = unused_ip_for_subnet(interface.subnet, interface.mac, interface.ip) if interface.subnet
       interface.ip6 = unused_ip_for_subnet(interface.subnet6, interface.mac, interface.ip6) if interface.subnet6
-      interface.save!
```

Comment:
This calls save twice, interfaces are actually saved when host is saved. At least according to my testing, this works both for quick provision and normal edit host form.

Reply:
Did we by any chance add this for the bond creation to work? I don't remember, need to check.

Reply:
Bond creation?

Reply:
I meant this: https://github.com/theforeman/foreman_discovery/blob/54f1724d8965dfd45eea3c0531df8ed95c509eea/app/services/foreman_discovery/import_hooks/lldp_neighbor.rb

But scratch that, it's unrelated.